sid/pathogenesis.py

```diff
@@ -31,21 +32,19 @@ def draw_course_of_disease(states, params, seed):
     """
     np.random.seed(next(seed))
 
-    states = states.copy()
```

Comment:
I do not think a copy is necessary. The data passed in by the user is not changed, because the first copy happens in `_process_initial_states`. From thereon, we can do whatever we want with the data.